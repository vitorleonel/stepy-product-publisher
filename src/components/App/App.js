import { HashRouter as Router } from 'react-router-dom';

import Navbar from '../Navbar';
import Steps from '../Steps';

const App = () => {
  return (
    <Router>
      <Navbar />
      <Steps />
    </Router>
  );
};

export default App;
