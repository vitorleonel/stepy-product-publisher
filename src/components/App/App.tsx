import Navbar from "../Navbar";
import Steps from "../Steps";

const App = (): JSX.Element => {
  return (
    <>
      <Navbar />
      <Steps />
    </>
  );
}

export default App;
