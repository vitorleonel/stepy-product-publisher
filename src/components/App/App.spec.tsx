import { render } from '@testing-library/react';

import App from './App';

describe('App Component', () => {
  test('should render correctly', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
