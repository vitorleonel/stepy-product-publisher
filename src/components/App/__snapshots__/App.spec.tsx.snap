// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render correctly 1`] = `
<div>
  <div
    class="bp3-navbar"
  >
    <div
      class="bp3-navbar-heading"
    >
      <img
        alt="Stepy"
        src="/images/brand.svg"
      />
      <h1>
        Product Publisher
      </h1>
    </div>
    <a
      href="https://github.com/vitorleonel/stepy-product-publisher"
      rel="noopener noreferrer"
      target="_blank"
    >
      <button
        class="bp3-button bp3-large bp3-minimal"
        type="button"
      >
        <span
          class="bp3-icon bp3-icon-git-repo"
          icon="git-repo"
        >
          <svg
            data-icon="git-repo"
            height="16"
            viewBox="0 0 16 16"
            width="16"
          >
            <desc>
              git-repo
            </desc>
            <path
              d="M5 9H4v1h1V9zm10-9H1C.45 0 0 .45 0 1v13c0 .55.45 1 1 1h3v1l2-1 2 1v-1h7c.55 0 1-.45 1-1V1c0-.55-.45-1-1-1zM4 13H2v-1h2v1zm10 0H8v-1h6v1zm0-2H3V2h11v9zM5 3H4v1h1V3zm0 4H4v1h1V7zm0-2H4v1h1V5z"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        <span
          class="bp3-button-text"
        >
          GitHub Repository
        </span>
      </button>
    </a>
  </div>
  <div
    class="bp3-progress-bar bp3-intent-success bp3-no-animation bp3-no-stripes"
  >
    <div
      class="bp3-progress-meter"
      style="width: 20%;"
    />
  </div>
  <div
    class="bp3-panel-stack2 bp3-panel-stack2-push steps"
  >
    <div
      class="bp3-panel-stack-view"
    >
      <section
        class="steps-item"
      >
        <div
          class="steps-item__header"
        >
          <h2
            class="bp3-heading"
          >
            Choose a category
          </h2>
          <p
            class="bp3-running-text bp3-text-large"
          >
            Choose a category that best represents your product to start the publish process.
          </p>
        </div>
        <div
          class="categories"
        >
          <div
            class="bp3-card bp3-interactive bp3-elevation-0 categories-item"
          >
            <h4
              class="bp3-heading"
            >
              Fashion
            </h4>
            <p
              class="bp3-running-text bp3-text-large"
            >
              T-shirts, pants, underwear and more.
            </p>
          </div>
          <div
            class="bp3-card bp3-interactive bp3-elevation-0 categories-item"
          >
            <h4
              class="bp3-heading"
            >
              Food
            </h4>
            <p
              class="bp3-running-text bp3-text-large"
            >
              Dairy, beverage and more.
            </p>
          </div>
          <div
            class="bp3-card bp3-interactive bp3-elevation-0 categories-item"
          >
            <h4
              class="bp3-heading"
            >
              Technology
            </h4>
            <p
              class="bp3-running-text bp3-text-large"
            >
              Cell phones, tablets, computers and more.
            </p>
          </div>
          <div
            class="bp3-card bp3-interactive bp3-elevation-0 categories-item"
          >
            <h4
              class="bp3-heading"
            >
              Vehicles
            </h4>
            <p
              class="bp3-running-text bp3-text-large"
            >
              Cars, trucks, motorcycle and more.
            </p>
          </div>
        </div>
      </section>
    </div>
  </div>
</div>
`;
